docs/guides/multi-platform.md

```diff
@@ -0,0 +1,235 @@
+
+You may also override the target platform when issuing the `earthly` build command. For example:
+
+```bash
+earthly --platform=linux/arm64 +build
+```
+
+This would cause the build to execute on the `linux/arm64` architecture.
+
+## Saving multi-platform images
+
+The easiest way to include platform information as part of a build is through the use of `FROM --platform`. For example:
+
+```Dockerfile
+FROM --platform=linux/arm/v7 alpine:3.11
+```
+
+If multiple targets create an image with the same name, but for different platforms, the images will be merged into a multi-platform image during export. For example:
+
+```Dockerfile
+build-all-platforms:
+    BUILD +build-amd64
+    BUILD +build-arm-v7
+
+build-amd64:
+    FROM --platform=linux/amd64 alpine:3.11
+    ...
+    SAVE IMAGE --push org/myimage:latest
+
+build-arm-v7:
+    FROM --platform=linux/arm/v7 alpine:3.11
+    ...
+    SAVE IMAGE --push org/myimage:latest
+```
+
+When `earthly --push +build-all-platforms` is executed, the build will push a multi-manifest image to the Docker registry. The manifest will contain two images: one for `linux/amd64` and one for `linux/arm/v7`. This works as such because both targets that save images use the exact same Docker tag for the image.
+
+Of course, in some situations, the build steps are the same (except they run on different platform), so the two definitions can be merged like so:
+
+```Dockerfile
+build-all-platforms:
+    BUILD --platform=linux/amd64 --platform=linux/arm/v7 +build
+
+build:
+    FROM alpine:3.11
+    ...
+    SAVE IMAGE --push org/myimage:latest
+```
+
+A more complete version of this example is available in [examples/multiplatform](https://github.com/earthly/earthly/tree/main/examples/multiplatform) in GitHub. You may try out this example without cloning by running
+
+```bash
+earthly github.com/earthly/earthly/examples/multiplatform:main+all
+docker run --rm earthly/examples:multiplatform
+docker run --rm earthly/examples:multiplatform_linux_amd64
+docker run --rm earthly/examples:multiplatform_linux_arm_v7
+```
+
+{% hint style='info' %}
+##### Note
+As of the time of writing this article, the `docker` CLI has limited support for working with multi-manifest images locally. For this reason, when exporting an image to the local Docker daemon, Earthly provides the different architectures as different Docker tags.
+
+For example, the above build would yield locally:
+
+* `org/myimage:latest`
+* `org/myimage:latest_linux_amd64` (the same as `org/myimage:latest` if running on a `linux/amd64` host)
+* `org/myimage:latest_linux_arm_v7`
+
+The additional Docker tags are only available for use on the local system. When pushing an image to a Docker registry, it is pushed as a single multi-manifest image.
+{% endhint %}
+
+## Creating multi-platform images without emulation
```

Comment:
It might be good to note this is only possible for languages that support cross complication